internal/planner/planner.go

```diff
@@ -160,7 +160,7 @@ func (p *Planner) buildFunctrie() error {
 	return nil
 }
 
-func (p *Planner) planRules(rules []*ast.Rule) (string, error) {
+func (p *Planner) planRules(rules []*ast.Rule, extras ...*ast.Term) (string, error) {
```

Comment:
This is a shortcut -- I didn't want to adjust all call sites, but only the one where I'd like to have extra args injected...